Remove a conditional that decides whether `quarto-preview.js`, the code which
handles hot-reloading when using `quarto preview`, should be built. In our
case, it should be, but the conditional was deciding not to.

IMPORTANT: This patch is temporary. A fix will be applied upstream to enable us
to remove this patch. See: 
https://github.com/conda-forge/quarto-feedstock/pull/36#issuecomment-1947515694 

IMPORTANT: Although the PR above is merged, this patch is still needed. See:
https://github.com/conda-forge/quarto-feedstock/issues/53

diff --git a/src/webui/quarto-preview/build.ts b/src/webui/quarto-preview/build.ts
index 55d4e4b6b..0663285fa 100644
--- a/src/webui/quarto-preview/build.ts
+++ b/src/webui/quarto-preview/build.ts
@@ -52,11 +52,8 @@ const buildFromGit = async () => {
 };
 
 // check if any of our repo files have a later time
-const build = buildFromArgs() || await buildFromGit();
-
-if (!build) {
-  console.log("No changes to quarto-preview.js, skipping build");
-  Deno.exit(0);
-}
+// Always build in conda-forge to ensure the file is present
+const build = true;
 
 console.log("Building quarto-preview.js");
 console.log("Installing...");
